File: 250129/007/007.py
class Class007:
    def __init__(self, code, location, time):
        self.code = code
        self.location = location
        self.time = time

code, location, time = input().split()
my_mission = Class007(code, location, time)

print("secret code :", my_mission.code)
print("meeting point :", my_mission.location)
print("time :", my_mission.time)

File: trail2/007/007.java
import java.util.*;

class Mission {
    String code;
    char point;
    int time;

    public Mission(String code, char point, int time) {
        this.code = code;
        this.point = point;
        this.time = time;
    }
}

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String code = sc.next();
        char point = sc.next().charAt(0);
        int time = sc.nextInt();

        Mission m = new Mission(code, point, time);
        System.out.println("secret code : " + m.code);
        System.out.println("meeting point : " + m.point);
        System.out.println("time : " + m.time);
    }
}